### PR TITLE
Added upcoming events for after the summer

### PR DESCRIPTION
**Pull request by ewels**

### CODE REVIEW

File: markdown/events/2021/bytesize-19-dsl2-pipeline-starter.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Bytesize 19: Where do you begin? DSL2 pipeline introduction"
+subtitle: Harshil Patel - Francis Crick Institiute, UK
```
Comment (drpatelh) [inline markdown/events/2021/bytesize-19-dsl2-pipeline-starter.md:3]:
```suggestion
title: "Bytesize 19: Where do I begin writing my own DSL2 pipeline?!"
subtitle: Harshil Patel - Seqera Labs, Spain
```

File: markdown/events/2021/bytesize-19-dsl2-pipeline-starter.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Join us for an episode of our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize 19: Where do you begin? DSL2 pipeline introduction
+
+This week, Harshil Patel ([@drpatelh](http://github.com/drpatelh/)) will present and introduction to developing pipelines in Nextflow DSL2.
```
Comment (drpatelh) [inline markdown/events/2021/bytesize-19-dsl2-pipeline-starter.md:21]:
```suggestion
This week, Harshil Patel ([@drpatelh](http://github.com/drpatelh/)) will present an introduction to developing pipelines in Nextflow DSL2 using nf-core community standards.
```

File: markdown/events/2021/bytesize-19-dsl2-pipeline-starter.md (hunk 1 above)
Comment (ewels) [reply markdown/events/2021/bytesize-19-dsl2-pipeline-starter.md:3]: ooh

**Review by drpatelh — APPROVED** [review]
:boom:

**Pull request merged**